Makefile.local.toml

```diff
@@ -54,6 +59,10 @@ args = ["clippy", "--all-targets", "--all-features", "--", "-D", "warnings"]
 category = "LOCAL USAGE"
 dependencies = ["cargo-check", "cargo-fmt", "cargo-fmt-unlinked", "cargo-clippy"]
 
+[tasks.check-wasm]
```

Comment:
Maybe these makefile changes are unrelated to this work and unused?

Reply:
Moved to https://github.com/surrealdb/surrealdb/pull/2796